internal/config/cgmgr/cgmgr_test.go

```diff
@@ -17,7 +17,7 @@ const (
 )
 
 // The actual test suite
-var _ = t.Describe("Config", func() {
+var _ = t.Describe("Cgmgr", func() {
```

Comment:
is this hunk related?

Reply:
not technically related but also trivial and self evident so I didn't bother adding a different commit